app/domain/operations/fdsh/ssa/h3/request_ssa_verification.rb

```diff
@@ -11,31 +11,89 @@ module H3
         class RequestSsaVerification
```

Comment:
I think we should do make some of this done only if ff is set, but we can discuss